packages/main/src/controllers/credentials/CredentialTypeController.ts

```diff
@@ -133,7 +137,7 @@ export class CredentialTypesController {
 
       const registrationResult = await agent.modules.anoncreds.registerCredentialDefinition({
         credentialDefinition: { issuerId, schemaId, tag: `${options.name}.${options.version}` },
-        options: { supportRevocation: true },
+        options: { supportRevocation: this.supportRevocation },
```

Comment:
See my comment above about making `supportRevocation` an option for credential type rather than a global setting.

Reply:
fix: add supportRevocation: boolean to CredentialTypeInfo